### PR TITLE
Add cart code type data import bundle

### PR DESCRIPTION
**Pull request by DerCalli**
- allow to use data import for importing cart code type (like voucher, gift card, ...)
- add unit tests
- ...

### CODE REVIEW

File: bundles/cart-code-type-data-import/composer.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  "description": "Cart code type data import module.",
+  "license": "MIT",
+  "authors": [
+    {
+      "name": "Daniel Rose",
+      "email": "daniel-rose@gmx.de"
+    }
+  ],
+  "require": {
+    "php": ">=7.3"
```
Comment (daniel-rose) [inline bundles/cart-code-type-data-import/composer.json:12]: Please check your dependencies...

File: bundles/cart-code-type-data-import/src/FondOfOryx/Zed/CartCodeTypeDataImport/Business/CartCodeTypeDataImportFacadeInterface.php [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+namespace FondOfOryx\Zed\CartCodeTypeDataImport\Business;
+
+use Generated\Shared\Transfer\DataImporterConfigurationTransfer;
+use Generated\Shared\Transfer\DataImporterReportTransfer;
+
+interface CartCodeTypeDataImportFacadeInterface
+{
+    /**
+     * Specification:
+     * - Imports Shipment Table Rate data.
```
Comment (daniel-rose) [inline bundles/cart-code-type-data-import/src/FondOfOryx/Zed/CartCodeTypeDataImport/Business/CartCodeTypeDataImportFacadeInterface.php:12]: Copy paste failure...

File: codeception.yml [hunk 3]
```diff
@@ -1,11 +1,9 @@
 namespace: fond-of-oryx/fond-of-oryx
 
 include:
-  - bundles/availability-alert
-  - bundles/availability-alert-cross-engage
-  - bundles/availability-alert-migrator
```
Comment (daniel-rose) [inline codeception.yml:6]: Why did you remove this line?

**Review by daniel-rose — CHANGES_REQUESTED** [review]

File: codeception.yml (hunk 3 above)
Comment (DerCalli) [reply codeception.yml:6]: oops, just a mistake

**Review by sourcecube — APPROVED** [review]

**Review by daniel-rose — APPROVED** [review]

**Pull request merged**